apps/pipelines/migrations/0006_pipelinechathistory_pipelinechatmessages.py

```
@@ -0,0 +1,44 @@
+                'unique_together': {('session', 'type', 'name')},
+            },
+        ),
+        migrations.CreateModel(
+            name='PipelineChatMessages',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_at', models.DateTimeField(auto_now_add=True)),
+                ('updated_at', models.DateTimeField(auto_now=True)),
+                ('human_message', models.TextField()),
+                ('ai_message', models.TextField()),
+                ('chat_history', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='messages', to='pipelines.pipelinechathistory')),
```

Ruff

Line too long (158 > 120)

(E501)